sopel/cli/config.py

```diff
@@ -22,11 +22,13 @@ def build_parser():
     # sopel-config list
     list_parser = subparsers.add_parser(
         'list',
-        help="List available configurations from Sopel's homedir",
+        help="List available configurations from Sopel's config directory",
         description="""
-            List available configurations from Sopel's homedir ({homedir})
-            with the extension "{ext}".
-        """.format(homedir=config.DEFAULT_HOMEDIR, ext='.cfg'))
+            List available configurations from Sopel's config directory
+            with the extension "{ext}". Use option --config-dir to use a
```

Comment:
Should we do this to align with changes in #1646?

```suggestion
            with the extension "{ext}". Use option ``--config-dir`` to use a
```